### ISSUE TITLE
Can fontawesomeicons have an alt property for SEO?

### ISSUE THREAD

**Issue by repejota** [vtalks/docs#12]
_From @repejota on February 6, 2018 3:29_

## Description

there is also something about role="link" aria-label="foo bar" that should be researched.

_Copied from original issue: vtalks/vtalks.net#71_

**Comment by repejota** [comment]
Looks like not.
